imports/util/email-invite.js

```diff
@@ -1,17 +1,38 @@
 import { Email } from 'meteor/email'
 import url from 'url'
+import bugzillaApi from '../util/bugzilla-api'
+const { callAPI } = bugzillaApi
 
-export const invite = (user, invitedBy) =>
-  Email.send({
-    to: user.emails[0].address,
-    from: process.env.FROM_EMAIL,
-    subject: `Invite by ${invitedBy.profile.name || invitedBy.emails[0].address}`,
-    text: `Hi there,
+export const invite = (user, invitedBy) => {
+  try {
+    const caseId = user.invitedToCases[0].caseId
+    const unitId = user.invitedToCases[0].unitId
+    const caseData = callAPI('get', `/rest/bug/${caseId}`, {}, true, true)
+    const unitData = callAPI('get', `/rest/product/${unitId}`, {}, true, true)
```

Comment:
These two API requests can and probably should run in parallel.
It can be done if you omit the last 'true' argument and it would return a promise and run in async mode.
Wrap both with Promise.all()